packages/react-scripts/template/README.md

```diff
@@ -217,7 +217,7 @@ In addition to [ES6](https://github.com/lukehoban/es6features) syntax features,
 * [Async/await](https://github.com/tc39/ecmascript-asyncawait) (ES2017).
 * [Object Rest/Spread Properties](https://github.com/tc39/proposal-object-rest-spread) (ES2018).
 * [Dynamic import()](https://github.com/tc39/proposal-dynamic-import) (stage 3 proposal)
-* [Class Fields and Static Properties](https://github.com/tc39/proposal-class-public-fields) (part of stage 3 proposal).
+* [Class Fields and Static Properties](https://github.com/tc39/proposal-class-fields) (part of stage 3 proposal).
```

Comment:
The new proposal is at stage 2.

Reply:
Are you sure ? I yes, I'm confused because I found this feature to be part of stage 3 both in the [tc39 proposal document](https://github.com/tc39/proposals#stage-3) and in the [Class field declarations for JavaScript page](https://github.com/tc39/proposal-class-fields).

Or maybe you are referring to the [Static class fields and private static methods](http://github.com/tc39/proposal-static-class-features/) proposal, which is listed in the [tc39 proposal document](https://github.com/tc39/proposals#stage-3) as in stage 3, but in [its own github readme](https://github.com/tc39/proposal-static-class-features/) as in stage 2...

Reply:
Are you sure ? If yes, I'm confused because I found this feature to be part of stage 3 both in the [tc39 proposal document](https://github.com/tc39/proposals#stage-3) and in the [Class field declarations for JavaScript page](https://github.com/tc39/proposal-class-fields).

Or maybe you are referring to the [Static class fields and private static methods](http://github.com/tc39/proposal-static-class-features/) proposal, which is listed in the [tc39 proposal document](https://github.com/tc39/proposals#stage-3) as in stage 3, but in [its own github readme](https://github.com/tc39/proposal-static-class-features/) as in stage 2...

PS: Sorry for the double notification, I originally posted this message with the wrong github account of mine :sweat_smile:

Reply:
Oops, look like the old proposal is at stage 2, which doesn't matter. 😅My bad!